ui/app/templates/components/secret-create-or-update.hbs

```diff
@@ -144,7 +144,7 @@
       <div class="control">
         <button
           type="submit"
-          disabled={{or @buttonDisabled this.validationErrorCount this.error}}
+          disabled={{or @buttonDisabled this.validationErrorCount}}
```

Comment:
I thought a while ago we decided to reserve disabling submit buttons for loading states only? but perhaps I'm misremembering 🤔

Reply:
You are probably correct, but this is a very old component. I can remove all the disabled states here unless there's a loading state we can use!

Reply:
My comment above posted without the second part 😞  - I meant to suggest that it might be best to just removing the disabled states all together - if you agree!

Reply:
I left passing `@buttonDisabled` from the parent, because the parent is actually tracking the model inflight status and disabling it from there prevents the save request from being fired twice